libhb/common.c

```diff
@@ -1283,7 +1324,10 @@ const char* const* hb_video_encoder_get_profiles(int encoder)
         case HB_VCODEC_X264:
             return hb_h264_profile_names;
 
-        case HB_VCODEC_X265:
+        case HB_VCODEC_X265_8BIT:
+        case HB_VCODEC_X265_10BIT:
+        case HB_VCODEC_X265_12BIT:
+        case HB_VCODEC_X265_16BIT:
             return hb_h265_profile_names;
 
```

Comment:
This is wrong, we need main10 for 10-bit, and whatever profiles exist and/or are supported by x265 for 12- and 16-bit.

So basically we need a profile names list for each bit depth.